### ISSUE TITLE
IS-08: Testing Outputs which don't include null in routable_inputs

### ISSUE THREAD

**Issue by garethsb-sony** [AMWA-TV/nmos-testing#531]
I don't have an implementation to test, but does the test suite correctly test making activations of Outputs which **don't** have `null` in their **/caps** `"routable_inputs"`?

In my understanding, unless `null` explicitly appears there, an Output channel cannot be set to `"input": null`?

@simonrankine, have I got that right?

**Comment by garethsb-sony** [comment]
Moreover, there seems to be a bug in `IS0801Test.test_13` that means it doesn't handle Outputs which **do** have `null` in their **/caps** `"routable_inputs"`. I'm seeing:

```
 * Running test_13
Traceback (most recent call last):
  File "...\nmos-testing\nmostesting\GenericTest.py", line 140, in execute_test
    self.result.append(method(test))
  File "...\nmos-testing\nmostesting\suites\IS0801Test.py", line 282, in test_13
    forbiddenRoutes.remove(routableInputID)
ValueError: list.remove(x): x not in list
```

**Comment by garethsb-sony** [comment]
There's a similar bug in `IS0801Test._test_01` since `TestHelper.compare_json` doesn't handle arrays that contain mixed types, the case in point here being, e.g.
```json
"routable_inputs": [ "input1", "input2", null ]
```

**Comment by garethsb-sony** [comment]
Hmm, I think `TestHelper.compare_json` may be broken in other ways because it uses `==`. I think it would end up comparing equal `{ "foo": true }` and `{ "foo": 1.0 }` because in Python `True == 1.0`.

**Comment by garethsb-sony** [comment]
I believe this is now fixed by #593. 🤞
